apps/site/assets/css/_footer.scss

```diff
@@ -58,38 +139,303 @@
+      padding-top: 24px;
+    }
+
+    @include media-breakpoint-up(lg) {
+      padding-bottom: 26px;
+    }
+
+  small {
+    font-size: 16px;
+    @include media-breakpoint-down(xs) {
+      white-space: nowrap;
+    }
+  }
+  
+  }
+
+  @include media-breakpoint-down(xs) {
+    .container {
+      padding-left: 4px;
+      padding-right: 4px;
+    }
+  }
+
+  @include media-breakpoint-only(md) {
+    font-size: $font-size-base-sm;
+    padding-bottom: $base-spacing * 2;
+  }
+
+  @include media-breakpoint-down(sm) {
+    padding-bottom: 50px;
+  }
+  .m-footer__links {
+    @include media-breakpoint-down(xs){
+      display: block;
+    }
+    @include media-breakpoint-up(lg){
+      display: block;
+
+    }
+  }
+  @include media-breakpoint-up(lg) {
+    margin-bottom: 0px;
+    padding-right: 40px;
+    font-size: $font-size-base;
+    .container {
+      ul {
+        display: flex;
+        flex-direction: column;
+        .m-footer__group {
+          width: 100%;
+        }
+      }
+    }
+
+    .m-footer__group {
+      padding-left: 15px;
+    }
   }
 
-  .m-footer--mobile__items {
-    text-align: center;
+  :nth-child(2) {
+    div {
+      @include media-breakpoint-up(lg) {
+        padding-left: 0px;
+      }
+    }
+    :first-child {
+      padding-top: 0px;
+    }
+    :nth-child(1) {
+      margin-top: 0;
+      margin-bottom: 0px;
+    }
+  }
 
-    @include not-last-child {
-      &::after {
-        color: $white;
-        content: '●';
-        margin-left: .25em;
-        margin-right: .25em;
+  .m-menu__feature_phone {
+    padding-top: 1rem;
+    a {
+      color: $brand-primary;
+    }
+    @include media-breakpoint-down(xs) {
+      display: flex;
+      a {
+        margin-top: 0px;
+        font-size: 14px;
+        display: inline;
+      }
+      strong {
+        padding-right: 4px;
+        font-size: 14px;
+        display: inline;
       }
     }
   }
+  @include media-breakpoint-up(lg) {
+    a {
+      padding-top: 12px;
+      margin-bottom: 0px;
+      display: inline;
+    }
+  }
+}
+
+.call-us-header {
+  @include media-breakpoint-up(lg) {
+    font-size:24px;
+    margin-top: 18px;
+    margin-bottom: 12px;
+    font-family: Inter UI,Helvetica Neue,Helvetica,Arial,sans-serif;
+  }
+  @include media-breakpoint-down(md) {
+    padding-left: 12px;
+  }
+  @include media-breakpoint-down(xs) {
+    margin-top: 2px;
+    font-size: 18px;
+    margin-bottom: 0px;
+  }
+}
+
+.numbers-container {
+  padding-top: 19px;
+  .m-footer__item {
+    display: flex;
+  }
+  strong {
+    margin-top: 12px;
+    padding-right: 4px;
+  }
+  a {
+    color: $brand-primary;
+  }
+}
+
+.all-container {
+  max-width: 100%;
+  @include media-breakpoint-up(lg) {
+    display: flex;
+  }
+}
 
-  .m-footer--mobile__item {
+.lower-footer {
+  @include media-breakpoint-down(xs) {
+    flex-direction: column;
+  }
+
+  @include media-breakpoint-up(lg) {
+    position: absolute;
+    margin: 0 40px 0 56px;
+    margin-right: 80px;
```

Comment:
These can be merged together. But also I don't think we need any `margin-right` here.

<img width="428" alt="image" src="https://user-images.githubusercontent.com/2136286/189410205-7a4a502f-f74d-4e9f-ac4a-22bb373b7663.png">